conformance/tests/httproute-rewrite-path.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: rewrite-path
+  namespace: gateway-conformance-infra
+spec:
+  parentRefs:
+  - name: same-namespace
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /prefix/one
+    filters:
+    - type: URLRewrite
+      urlRewrite:
+        path:
+          type: ReplacePrefixMatch
+          replacePrefixMatch: /one
+    backendRefs:
+    - name: infra-backend-v1
+      port: 8080
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /full/one
+    filters:
+    - type: URLRewrite
+      urlRewrite:
+        path:
+          type: ReplaceFullPath
+          replaceFullPath: /one
+    backendRefs:
+    - name: infra-backend-v1
```

Comment:
I believe this should be `infra-backend-v2` given https://github.com/LiorLieberman/gateway-api/blob/7dc2bcaed9f75a4cf7e6908c4b9262b6c3609fec/conformance/tests/httproute-rewrite-path.go#L66

Reply:
good catch, thanks! 

opened https://github.com/kubernetes-sigs/gateway-api/pull/1628 to fix it